src/roles/vsd-deploy/tasks/main.yml

```diff
@@ -373,7 +374,8 @@
       debug: var=vsd_proc verbosity=1
 
     when: monit_mail_server is defined
-
+  
+  when: not skip_vsd_deploy
```

Comment:
What testing have you performed on this?  Have you tried running vsd deploy on an already running setup?  Have you done it both with HA and SA?  Also, I think we should run an upgrade test to make sure that an upgrade doesn't mistakenly have the deploy step skipped.

Reply:
I have run INSTALL-GCP-SA and then run the metroae install vsd on the testbed. However, I haven't tested it on HA I will test on it and also test for an upgrade too.